lib/publiccloud/provider.pm

```diff
@@ -452,14 +452,9 @@ sub terraform_apply {
     my $ips;
     my $resource_id;
     if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
-        foreach my $vm_type ('cluster_nodes', 'drbd', 'netweaver') {
-            if (is_azure && $vm_type eq 'cluster_nodes') {
-                push @{$vms}, @{$output->{$vm_type . '_name'}->{value}[0]};
-                push @{$ips}, @{$output->{$vm_type . '_public_ip'}->{value}[0]};
-            } else {
-                push @{$vms}, @{$output->{$vm_type . '_name'}->{value}};
-                push @{$ips}, @{$output->{$vm_type . '_public_ip'}->{value}};
-            }
+        foreach my $vm_type ('hana_public', 'drbd', 'netweaver') {
```

Comment:
Seems this has to be:
```
    foreach my $vm_type ('hana', 'drbd', 'netweaver') {
```
It failed on the GCP VR: http://1b124.qa.suse.de/tests/175#step/sles4sap/125